packages/truffle-contract/lib/execute.js

```diff
@@ -84,11 +85,11 @@ var execute = {
 
   /**
    * Executes method as .call and processes optional `defaultBlock` argument.
-   * @param  {Function} fn      method
-   * @param  {Array}    inputs  ABI segment defining the methods inputs.
-   * @return {Promise}          Return value of the call.
+   * @param  {Function} fn         method
+   * @param  {Array}    methodABI  ABI segment defining the methods inputs.
```

Comment:
This should change to `{Object}` I think.

One note: do we want to check for whether it's an array, to accept the array format too?